examples/tests/AuctionSpec.hs

```diff
@@ -0,0 +1,219 @@
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module AuctionSpec where
+
+import qualified Auction as A
+import qualified Auction.Offchain as A
+import Control.Applicative
+import Control.Arrow
+import Cooked.MockChain
+import Cooked.Tx.Constraints
+import Data.Default
+import qualified Data.Map.Strict as M
+import qualified Ledger as L
+import qualified Ledger.Ada as Ada
+import qualified Ledger.Value as Value
+import Test.Tasty
+import Test.Tasty.HUnit
+
+-- * Parameters and initial distributions
+
+-- Just so we have something to sell in our auction that's not Ada:
+-- Have a banana.
+
+bananaAssetClass :: Value.AssetClass
+bananaAssetClass = Value.assetClass (Value.currencySymbol "f00d") (Value.tokenName "Banana")
```

Comment:
You can use `permanent` or `quick` values too; those are handled in a special by cooked and come from the [`Currencies`](https://github.com/tweag/plutus-libs/blob/main/cooked-validators/src/Cooked/Currencies.hs) module:

```haskell
import Cooked.Currencies

bananaAssetClass :: Value.AssetClass
bananaAssetClass = permanentAssetClass "Banana"
```